frontend/public/components/_overview.scss

```diff
@@ -26,29 +26,91 @@
 }
 
 .overview {
-  width: 100%;
+  bottom: 0;
   display: flex;
-}
+  left: 0;
+  overflow-y: hidden;
+  position: absolute;
+  right: 0;
+  top: 0;
 
-.overview__body {
-  flex-grow: 1;
-}
+  .overview__main-column {
+    height: 100%;
+    overflow: hidden;
+    overflow-y: auto;
+    width: 100%;
+    -webkit-overflow-scrolling: touch;
+
+    .overview__main-column-section {
+      flex-grow: 1;
```

Comment:
`.overview__main-column` isn't a flex container, so I don't think this is needed?